extract_endpoint/src/extract_endpoint/endpoint.py

```diff
@@ -78,11 +80,9 @@ def upload_file() -> typing.Tuple[str, int]:
         flask.abort(HTTPStatus.BAD_GATEWAY)
 
     timestamp = flask.request.form.get('timestamp', None)
```

Comment:
You can just use `flask.request.form['timestamp']` here since you've already checked for it above.